content/docs/command-reference/diff.md

```diff
@@ -107,7 +107,8 @@ $ dvc diff
 
 Let's checkout the
 [3-add-file](https://github.com/iterative/example-get-started/releases/tag/3-add-file)
-tag, corresponding to the [Add Files](/doc/tutorials/get-started/add-files) _Get
+tag, corresponding to the
+[tracking data](/doc/tutorials/get-started/data-versioning#tracking-data) _Get
```

Comment:
This is causing CI link check to fail. Shoudn't. But why only this one? There's many other such md links in the changes and throughout the docs. Cc @casperdcl thanks

Reply:
what do you mean? Ca n'existe pas. "The sadness."

![image](https://user-images.githubusercontent.com/10780059/81999566-5de29500-964d-11ea-9a84-39e49a5ffead.png)

Reply:
Yes, new file. There's a bunch of instances of this one and other new files linked in these changes though. That's what seems strange to me this time.

Reply:
interesting. There used to be an issue where only the first failing link per file was reported but I thought I'd fixed that.

Reply:
🤷‍♂️ 🙂